Difi.SikkerDigitalPost.Klient/Difi.SikkerDigitalPost.Klient.csproj

```diff
@@ -60,11 +60,19 @@
       <HintPath>..\packages\difi-felles-utility-dotnet.0.8.0.27738\lib\net45\Difi.Felles.Utility.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
Den brukes ikke eksplisitt av oss, men er en avhengighet for `HttpClientFactory` og kommer derfor med. Litt tjafsete, men slik må det nesten bare bli ...